src/game/AI/ScriptDevAI/base/BossAI.h

```diff
@@ -56,6 +56,11 @@ class BossAI : public CombatAI
         void JustDied(Unit* killer = nullptr) override;
         void JustReachedHome() override;
         void Aggro(Unit* who = nullptr) override;
+        void Reset() override
```

Comment:
please ignore this for now. `meson` is my personal development branch, so some commits that I use for my own reasons may land in this PR. I'll sort them out if ever any actual interest in merging meson support becomes apparent

Reply:
nope, won't ignore it, can't make me